code/parachain/frame/staking-rewards/src/lib.rs

```diff
@@ -1782,6 +1820,60 @@ pub(crate) fn claim_of_stake<T: Config>(
 }
 
 impl<T: Config> Pallet<T> {
+	pub fn register_share_asset(
+		staked_asset_id: T::AssetId,
+		nonce: u64,
+	) -> Result<T::AssetId, DispatchError> {
+		// Create name as "x[STAKED_TOKEN_NAME]"
+		let mut staked_asset_name = T::AssetsTransactor::name(staked_asset_id);
+		let mut name: Vec<u8> = vec![b'x'];
```

Comment:
if staked_asset_id doesnt have a name or symbol, would a new local asset name be informative?

Reply:
Name has been removed, we can append fitting names and symbols if needed later